helpers/generate.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function generate() {
+  pip3 install --user -r ./helpers/generate_modules/requirements.txt
+  ./helpers/generate_modules/generate_modules.py
+}
+
+function check_generate() {
```

Comment:
Why was this function rewritten to be different from master? I'd like an explanation of why these changes were done.

Reply:
1) At the moment we don't have function to generate submodules from `autorgen` directory in docker-image. 
2) I'd changed function `find_files` (added [path `./autogen`](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/262/files#diff-d87a044699e75ebf31488eecb4df54ccR58)) to exclude this dir from terraform format checks